src/dso_api/dynamic_api/remote/views.py

```diff
@@ -145,10 +162,10 @@ def _raise_http_error(self, response: HTTPResponse):  # noqa: C901
                 )
             else:
                 raise BadGateway(detail_message)
-        elif response.status == 403:
+        elif response.status == 403:  # "forbidden"
```

Comment:
Hier zou je nog de constanten uit https://docs.python.org/3/library/http.html kunnen gebruiken.

Reply:
Ja, ik zat zelf ook `status.HTTP_403_FORBIDDEN` te overwegen uit het rest framework, maar op de een of andere manier vond ik dit toch duidelijker.